ports/esp8266/esp_mphal.h

```diff
@@ -24,6 +24,9 @@
  * THE SOFTWARE.
  */
 
+#ifndef _ESP8266_ESP_MPHAL_H
+#define _ESP8266_ESP_MPHAL_H
```

Comment:
please use `MICROPY_INCLUDED_ESP8266_ESP_MPHAL_H`

Reply:
Dropped this commit entirely, but I'll submit this as it's own small PR

Reply:
Over here now: https://github.com/micropython/micropython/pull/13147